CrazyCanvas/Source/GUI/KillFeedGUI.cpp

```diff
@@ -0,0 +1,82 @@
+#include "GUI/KillFeedGUI.h"
+
+#include "NoesisPCH.h"
+#include "Log/Log.h"
+
+#include "Teams/TeamHelper.h"
+
+
+KillFeedGUI::KillFeedGUI() :
+	m_KillFeedTimers()
+{
+	Noesis::GUI::LoadComponent(this, "KillFeedGUI.xaml");
+}
+
+KillFeedGUI::~KillFeedGUI()
+{
+}
+
+void KillFeedGUI::InitGUI()
+{
+	//m_pKillFeedStoryBoard = FindResource<Noesis::Storyboard>("KillFeedStoryBoard");
+	//m_pKillFeedDoubleAnimation = FindName<Noesis::DoubleAnimation>("KillFeedAnimation");
```

Comment:
If you want to work on the transparency part later then I guess you can keep the comments.

Reply:
Yeah no, what we have now is fully functional and enough for now. =) I'll remove these